helm_tests/airflow_aux/test_pod_template_file.py

```diff
@@ -660,6 +660,26 @@ def test_safe_to_evict_annotation(self, safe_to_evict: bool):
             "cluster-autoscaler.kubernetes.io/safe-to-evict": "true" if safe_to_evict else "false"
         }
 
+    def test_safe_to_evict_annotation_other_services(self):
+        docs = render_chart(
+            values={
+                "workers": {"safeToEvict": False},
+                "scheduler": {"safeToEvict": True},
```

Comment:
Can you add a comment on why you are setting config for scheduler/triggerer/etc, and checking those deployments too, in the pod template file tests?

Or, maybe just remove those and leave it worker/pod template? That might be better.

Reply:
Right, I didn't quite connect the dots on that. Can you add that (feel free to shorten it a bit) as a comment/docstring on this test so there is some context right there for future contributors?

Reply:
Sorry for my previous message, Probably I should have put it like this, one consequence of this issue is that if you apply the rendered manifest of the chart in kubernetes, workers' safeToEvict always overwrites safeToEvict of other services. This test covers that.

I just added a docstring to the test function hoping it would be both short and descriptive, please let me know if there were a need for any adjustments

Reply:
Perfect, thank you!